input/tex/extensions/ams.rst

```diff
@@ -67,12 +67,12 @@ following values:
 .. _tex-ams-multlineWidth:
 .. describe:: multlineWidth: '100%'
 
-The width to use for multline environments.
+The width to use for multiline environments.
```

Comment:
The environment name really is `multline`, not `multiline`, so this needs to be left as is.

Reply:
```suggestion
The width to use for multline environments.
```